tests/periph_pm/main.c

```diff
@@ -28,10 +28,20 @@
 #include "periph/rtc.h"
 #endif
 #include "pm_layered.h"
+extern int _pm(int argc, char **argv);
 #endif
 #include "shell.h"
 
 #ifdef MODULE_PM_LAYERED
+
+/* TODO deduplicate this definition with the one in sys/pm_layered/pm.c */
```

Comment:
You could move the definition to `pm_layered.h`. Is there a reason that prevents it ?

Reply:
I don't think so. @kaspar030 what do you think? Could we integrate this into #13380 as well?